### PR TITLE
global: add missing project files

### LINKED ISSUE #2: Add missing project files
Similar to inveniosoftware/cookiecutter-invenio-module we need:

```
.editorconfig
.gitignore
.travis.yml
AUTHORS.rst
LICENSE
README.rst
cookiecutter.json
docs
hooks
run-tests.sh
```

### PR DESCRIPTION
**Pull request by ChiaraBi**
* close #2

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2015-2018 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+==============================
```
Comment (dinosk) [inline README.rst]: needs more `=`

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  <https://coveralls.io/>`_.
+- **Your toolchain:** Ignores a decent set of files when working with Git and
+  `Docker <https://www.docker.com/>`_. Gets your editor to adapt project
+  guidelines by providing an `EditorConfig <http://editorconfig.org/>`_ file.
+
+Configuration
+-------------
+To generate correct files, please provide the following input to Cookiecutter:
+
+==================== =============================================
```
Comment (dinosk) [inline README.rst]: The input for the datamodel:
```
project_name [My datamodel]: 
project_shortname [my-datamodel]: 
package_name [my_datamodel]: 
github_repo [my-datamodel/my-datamodel]: 
description [Invenio data model.]: 
author_name [CERN]: 
author_email [info@inveniosoftware.org]: 
year [2018]: 
copyright_holder [CERN]: 
Select elasticsearch_version:
1 - v6
2 - v5
Choose from 1, 2 [1]: 
extension_class [Mydatamodel]: 
pid_name [id]:
```

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+.. image:: https://img.shields.io/coveralls/inveniosoftware/cookiecutter-invenio-datamodel.svg
+        :target: https://coveralls.io/r/inveniosoftware/cookiecutter-invenio-datamodel
+
+.. image:: https://img.shields.io/pypi/v/cookiecutter-invenio-datamodel.svg
+        :target: https://pypi.org/pypi/cookiecutter-invenio-datamodel
+
+This `Cookiecutter <https://github.com/audreyr/cookiecutter>`_ template is
+designed to help you to bootstrap an `Invenio
+<https://github.com/inveniosoftware/invenio>`_ service. It offers you the
```
Comment (dinosk) [inline README.rst]: maybe rephrase to: "help you bootstrap the data model for your Invenio service"

File: .editorconfig [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
+
+# Python files
+[*.py]
+indent_size = 4
+# isort plugin configuration
+known_first_party = invenio
```
Comment (switowski) [inline .editorconfig]: Should this `known_first_party` be here? I would remove it, as this is not really the `invenio` repo.

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+python:
+  - "2.7"
+  - "3.5"
+  - "3.6"
+
+before_install:
+  - "travis_retry pip install --upgrade pip"
+  - "travis_retry pip install cookiecutter"
+  - "travis_retry pip install Sphinx>=1.4.2 sphinx_rtd_theme>=0.1.7"
```
Comment (slint) [inline .travis.yml]: This theme (`sphinx_rtd_theme`) is not actually being used, so it can be removed

File: .editorconfig (hunk 4 above)
Comment (ChiaraBi) [reply .editorconfig]: I have seen that `known_first_party` is present also in cookiecutter-invenio-instance and cookiecutter-invenio-module. Should we remove it also from there?

File: README.rst [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+`description`           A short description of the functionality of the module,
+                        its length should not extend one line.
+`author_name`           The name of the primary author of the project, not
+                        necessarily the same as the copyright holder.
+`author_email`          E-Mail address of the primary author.
+`year`                  Current year.
+`copyright_holder`      Name of the person or organization who acts as the
+                        copyright holder of this project.
+`elasticsearch_version` The version of ElasticSearch you are planning to use.
+                        Versions 5.x (`1`) and 6.x (`2`)
```
Comment (slint) [inline README.rst]: The actual parameters are `v5` and `v6` (e.g. when using from the CLI `cookiecutter ... elasticsearch_version=v6`).

File: .editorconfig (hunk 4 above)
Comment (slint) [reply .editorconfig]: @lnielsen `cookiecutter-invenio-module` is actually a bit outdated (e.g. it still has GPL2 license). Should we deal with it in this Sprint? I think it's important unless we advise people to develop modules inside their instance instead...

File: README.rst (hunk 2 above)
Comment (dinosk) [reply README.rst]: Yes the description as you wrote it in the last changes I think is pretty clear
